ranger/ext/popen_forked.py

```diff
@@ -27,5 +27,5 @@ def Popen_forked(*args, **kwargs):  # pylint: disable=invalid-name
             Popen(*args, **kwargs)  # pylint: disable=consider-using-with
         os._exit(0)  # pylint: disable=protected-access
     else:
-        os.wait()
+        os.waitpid(pid, 0)
```

Comment:
Only difference that's obvious to me from the docs is throwing an exception on Windows. What's the rationale here?